Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16-alpine
+WORKDIR /app
+COPY . .
+RUN npm ci --only=production
+EXPOSE 80
```

Comment:
443 should be exposed as well, as the main purpose of https-front is to serve "raw" HTTPS.

Reply:
Done